ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/activity-picklist-remote-auto-complete-options.component.ts

```diff
@@ -85,15 +87,17 @@ export class ActivityPicklistRemoteAutoCompleteOptionsComponent
 
     ngOnInit(): void {
         this.picklistOptions$ = this.searchValue$.pipe(
-            debounceTime(500),
+            debounceTime(1000),
```

Comment:
1 second is kind of a long time! stay with 500 or even go down to 300. But you can try it and set a value that makes sense experimentally.

Reply:
Reverted to 500